### PR TITLE
Change global_ocean ISC test cases to use BedMachine

### PR DESCRIPTION
**Pull request by xylar**
This merge adds support for a new topography for runs with ice-shelf cavities (wISC).  The new topography combines BedMachineAntarctica and the GEBCO global topography.  The combined data set is on a 1/20 degree grid (2x higher res than the previous topography based on Bedmap2 and ETOPO2).

### CODE REVIEW

**Comment by xylar** [comment]
I successfully tested QU240wISC on my laptop.  I will test this and the other 2 wISC test cases on IC tomorrow.

**Comment by xylar** [comment]
A companion PR updates geometric_features with the appropriate coastline: https://github.com/MPAS-Dev/geometric_features/pull/126

I am testing with a local build of the geometric_features package and manually copying the new coastline features into my `geometric_data` directory.

**Comment by xylar** [comment]
## Testing

I'm testing the following on Grizzly
* `QU240wISC`
  * init - successful!
* `EC60to30wISC`
  * init - successful!
  * spin_up - underway
* `SO60to10wISC`
  * init - underway
  * spin_up - ?

**Comment by xylar** [comment]
## Example: EC60to30wISC

Here is the ice-shelf mask from the `EC60to30wISC` test case:
ice_shelf_mask

File: testing_and_setup/compass/ocean/global_ocean/CUSP12/spin_up/config_driver.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<driver_script name="run_test.py">
+<driver_script name="run.py">
```
Comment (xylar) [inline testing_and_setup/compass/ocean/global_ocean/CUSP12/spin_up/config_driver.xml]: This and similar lines in other `spin_up` driver scripts seem like they should be changed as above to be consistent with `init` driver scripts.  @mark-petersen, let me know if you disagree.

**Comment by xylar** [comment]
@kristin-hoch, are you around and are you willing/interested in testing your latest setup with this new bathymetry?  If so, maybe stop by my office.

**Comment by xylar** [comment]
## SO60to10wISC

Ice shelves:
SO60to10_shelves
Bathymetry:
SO60to10_bathy

Zooming in on the area where SSH adjustment fails, problem cell is isolated single-cell ice shelf near the center.
Bathymetry:
SO60to10_bathy_problem
Ice draft:
SO60to10_draft_problem

I think the problem would be helped by smoothing.  Either of the source bathymetry file or during init.  See #436

**Comment by mark-petersen** [comment]
@xylar this is a good time to rebase this PR on `ocean/develop`. All recent changes should be orthogonal, but it's easier to rebase now.

**Comment by xylar** [comment]
@mark-petersen, I'll do that.  I'm adding smoothing of topography and will rebase that onto `ocean/devleop` and then this onto that.

**Comment by xylar** [comment]
## EC60to30wISC bathymetry at 3 levels of smoothing

No smoothing:
bot_smooth0

3 iterations of smoothing:
bot_smooth3

6 iterations of smoothing:
bot_smooth6

I propose to run G-cases with each of these to compare the effects.

**Comment by mark-petersen** [comment]
Please rebase later today or tomorrow.

**Comment by xylar** [comment]
@mark-petersen, this was just rebased onto #473.  I'm not 100% sure I got the merge right.  If not, the place where problems would emerge is in creating the mesh without ice-shelf cavities (or creating mapping files from that mesh).  We can (and should!) test that out.

**Comment by mark-petersen** [comment]
OK. I'll test the nightly regression suite, and also the QU240wISC to be sure.

File: testing_and_setup/compass/ocean/global_ocean/CUSP12/spin_up/config_driver.xml (hunk 1 above)
Comment (mark-petersen) [reply testing_and_setup/compass/ocean/global_ocean/CUSP12/spin_up/config_driver.xml]: Agreed. Easier to be consistent and only use run.py

**Comment by mark-petersen** [comment]
Note: We need to change to this bathymetry for all the meshes in another PR.

**Pull request merged**